create_test_database.py

```diff
@@ -72,5 +74,17 @@ def prep_instance():
     created_op = database.create()
     created_op.result(120)
 
+    config = configparser.ConfigParser()
+
+    config.read("setup.cfg")
+    url = "spanner:///projects/{project}/instances/{instance_id}/databases/compliance-test".format(
+        project=PROJECT, instance_id=instance_id
+    )
+    config.set("db", "default", url)
+
+    with open("setup.cfg", "w") as configfile:
```

Comment:
Another solution is to create `test.cfg` file and write db_url every time which override the db tag of `setup.cfg ` file.

Reply:
I do like the option of creating a `test.cfg` file where [db] -> default URL will be dynamically updated with every test run.